### PR TITLE
Homework_loops

### PR DESCRIPTION
**Pull request by Pticatos86**
all tasks has been done, my brain is exhausted ), but i am happy! Loops are difficult for understanding. I need more practice in loops

### CODE REVIEW

File: Loops/loops_homework_6.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+list_some_elements = [23, 0, 56, 100, 50, True, False, 'abc', 10.5]
+client_num = int(input("Please, enter your number: "))
+dividing_list_num = []
+for elements in list_some_elements:
+    if type(elements) == int:
+        dividing_num = elements // client_num
+        dividing_list_num.append(dividing_num)
+
```
Comment (MarinaZelenska) [inline Loops/loops_homework_6.py:8]: I am not sure that your decision is valid
Should be like this
image

File: Loops/loops_homework_6.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+print(f"After dividing your number {client_num} by numbers from list {list_some_elements} "
+      f"you get numbers in list {dividing_list_num}")
+
+
+unsorted_list = [1, 5, 68, 0]
+for num in range(1, len(unsorted_list)):
+    if unsorted_list[num] != unsorted_list[num - 1] + 1:
+        print(f" First inconsistent number in list {unsorted_list} is {unsorted_list[num]}")
+        break
+
```
Comment (MarinaZelenska) [inline Loops/loops_homework_6.py:20]: You forgot about else block(
image

File: Loops/loops_homework_6.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+print()
+print()
+
+
+# I looked up this solution of task 3 in internet, i tried on my own way, but not correct
+figure_num = 6
+for raw in range(1, figure_num):
+    for column in range(1, raw + 1):
+        print(column, end=' ')
+    print()
```
Comment (MarinaZelenska) [inline Loops/loops_homework_6.py:26]: Yes, this task should be solved in inner loop
But, I am very glad that you try. Never give up and try try try .....

File: Loops/loops_homework_6.py (hunk 1 above)
Comment (Pticatos86) [reply Loops/loops_homework_6.py:8]: fixed

File: Loops/loops_homework_6.py (hunk 2 above)
Comment (Pticatos86) [reply Loops/loops_homework_6.py:20]: add else block

File: Loops/loops_homework_6.py (hunk 3 above)
Comment (Pticatos86) [reply Loops/loops_homework_6.py:26]: delete my code with lists, remain code with inner loop

**Review by MarinaZelenska — APPROVED** [review]

**Pull request merged**